kotlin-yaml/src/test/kotlin/com/baeldung/yaml/yamlkt/YamlktUnitTest.kt

```diff
@@ -0,0 +1,262 @@
+package com.baeldung.yaml.yamlkt
+
+import com.baeldung.yaml.verifyUsers
+import kotlinx.serialization.decodeFromString
+import org.junit.jupiter.api.Test
+import kotlin.test.assertEquals
+import kotlin.test.assertTrue
+import net.mamoe.yamlkt.Yaml
+import net.mamoe.yamlkt.YamlMap
+import net.mamoe.yamlkt.YamlNull
+import net.mamoe.yamlkt.getPrimitive
+
+class YamlktUnitTest {
+
+    private val yamlDefault = Yaml.Default
```

Comment:
Let's use a fully qualified import to make it clear which library we are importing from

Reply:
Fixed